src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -151,6 +152,30 @@ internal string Arguments
 
         private readonly StringBuilder _arguments = new StringBuilder();
 
+        internal string[] ArgumentList
```

Comment:
It doesn't appear that `ArgumentList` couldn't stay a List instead of being an array?

Reply:
the API takes a string array, so I thought it would be better for me to do it this rather than calling the `ToArray` inline.